pkg/xds/generator/outbound_proxy_generator.go

```diff
@@ -101,6 +101,10 @@ func (_ OutboundProxyGenerator) generateLDS(proxy *model.Proxy, subsets []envoy_
 				Configure(envoy_listeners.Tracing(proxy.TracingBackend)).
 				Configure(envoy_listeners.HttpAccessLog(meshName, envoy_listeners.TrafficDirectionOutbound, sourceService, serviceName, proxy.Logs[serviceName], proxy)).
 				Configure(envoy_listeners.HttpOutboundRoute(envoy_names.GetOutboundRouteName(serviceName)))
+		case mesh_core.ProtocolUDP:
+			filterChainBuilder.
+				Configure(envoy_listeners.UdpProxy(serviceName, subsets[0])).
```

Comment:
Please add a comment here why we are choosing only the first subset (lack of support of subset loadbalancing for UDPProxy in Envoy)